src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Ball extends ColorSupplier {
```

Comment:
```suggestion
public class Ball {
```
you should create a global ColorSupplier field inside of the class where you use it